hazelcast-jet-core/src/main/java/com/hazelcast/jet/Sources.java

```diff
@@ -136,10 +136,11 @@ private Sources() {
      * com.hazelcast.projection.Projections#singleAttribute(String)
      * singleAttribute()} and {@link
      * com.hazelcast.projection.Projections#multiAttribute(String...)
-     * multiAttribute()}), depending on how the data is stored in the map.
-     * Similarly, using the {@link com.hazelcast.query.PredicateBuilder
-     * PredicateBuilder} will build a predicate that makes use of any indexes
-     * created on the map.
+     * multiAttribute()}), depending on the {@link com.hazelcast.config.InMemoryFormat}
```

Comment:
I'm not sure if this is a good way to describe it since `InMemoryFormat` is either `BINARY` or `OBJECT` when using OSS HZ. With `OBJECT` you always have the advantage (even with plain lambdas) and with `BINARY` you may not get it either way (if your object doesn't use portable serialization).